db/migrate/20200328094815_acts_as_taggable_on_migration.acts_as_taggable_on_engine.rb

```diff
@@ -1,13 +1,13 @@
 # This migration comes from acts_as_taggable_on_engine (originally 1)
 class ActsAsTaggableOnMigration < ActiveRecord::Migration[6.0]
   def self.up
-    create_table ActsAsTaggableOn.tags_table do |t|
```

Comment:
Should all of these old migrations have been changed?

Reply:
Ah, I suppose since we are removing the gem, they have to be.  Will this work fine on production then?

Reply:
@jamiebuckley this is @timwis changes.